app/models/review.rb

```diff
@@ -0,0 +1,42 @@
+class Review < ActiveRecord::Base
+  belongs_to :appointment # связь с должностями
+  belongs_to :university, foreign_key: :university_id
+  belongs_to :university_author, class_name: University, foreign_key: :university_auth_id
+
+  #validates :contract_date, presence: true
+  #validates :sheet_number, presence: true
+
+  enum status: {    # статус рецензии
+      обработка: 0,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.